packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -509,77 +517,49 @@ describe("editable-tree: read-only", () => {
 
 	it("primitives are unwrapped at root", () => {
 		const rootSchema = fieldSchema(FieldKinds.value, [int32Schema.name]);
-		const schemaData: SchemaData = {
-			treeSchema: schemaMap,
-			globalFieldSchema: new Map([[rootFieldKey, rootSchema]]),
-		};
-		const forest = setupForest(schemaData, [{ type: int32Schema.name, value: 1 }]);
+		const schemaData: SchemaData = buildTestSchema(rootSchema);
+		const forest = setupForest(schemaData, 1);
 		const context = getReadonlyEditableTreeContext(forest);
 		assert.equal(context.unwrappedRoot, 1);
 		expectFieldEquals(forest.schema, context.root, [{ type: int32Schema.name, value: 1 }]);
 		context.free();
 	});
 
 	it("primitives under node are unwrapped, but may be accessed without unwrapping", () => {
-		const rootSchema = fieldSchema(FieldKinds.value, [optionalChildSchema.name]);
-		const schemaData: SchemaData = {
-			treeSchema: schemaMap,
-			globalFieldSchema: new Map([[rootFieldKey, rootSchema]]),
-		};
-		const forest = setupForest(schemaData, [
-			{
-				type: optionalChildSchema.name,
-				fields: { child: [{ type: int32Schema.name, value: 1 }] },
-			},
-		]);
+		const parentSchema = TypedSchema.tree("parent", {
+			local: { child: TypedSchema.field(FieldKinds.value, stringSchema) },
+		});
+		const rootSchema = TypedSchema.field(FieldKinds.value, parentSchema);
+		const schemaData: SchemaData = SchemaAware.typedSchemaData(
+			new Map([[rootFieldKey, rootSchema]]),
+			stringSchema,
+			parentSchema,
+		);
+		const forest = setupForest(schemaData, { child: "x" });
 		const context = getReadonlyEditableTreeContext(forest);
 		assert(isUnwrappedNode(context.unwrappedRoot));
-		assert.equal(context.unwrappedRoot["child" as FieldKey], 1);
+		assert.equal(context.unwrappedRoot["child" as FieldKey], "x");
 
 		// access without unwrapping
 		const child = context.unwrappedRoot[getField](brand("child"));
 		assert(isEditableField(child));
-		expectFieldEquals(forest.schema, child, [{ type: int32Schema.name, value: 1 }]);
-		context.free();
-	});
-
-	it("undefined values not allowed", () => {
```

Comment:
This test is problematic since its it requires constructing an invalid (out of schema tree) in a way that fails with the new APIs.

This test is fine to delete since what it is testing for isn't useful behavior to ensure: it is testing that when already in an invalid state (out of schema) reading of the invalid data crashes in a specific way. What we actually want to ensure long term is that we can't get into invalid states, not how they crash.